pages/projects/allen.md

```diff
@@ -0,0 +1,18 @@
+---
+permalink: /projects/allen.html
+layout: project
+title: GPU Trigger Project
+shortname: allen
+pagetype: project
+image: logos/allen.png
+blurb: Allen: a GPU trigger for LHCb
+focus-area: ia
+team:
+ - mityinzer
+ - dcraik
+ - mdsokoloff
+---
+
+We are participating in the development of a GPU-based trigger, which would replace the first-level trigger in the LHCb upgrade. 
+A paper describing the project can be found [here](https://arxiv.org/abs/1912.09161) and the code is available in [the gitlab repository](https://gitlab.cern.ch/lhcb/Allen).
```

Comment:
Can't you just add this as a paper in `/_data/publications` and mark it with `project: allen`? Then it will show up on this page automatically.

Reply:
Edit: Looks like you added the publication so the part about the paper here probably should be removed.